api/graphql/schema.graphql

```diff
@@ -1184,6 +1184,48 @@ input GenericJobTitleBelongsToMany {
     sync: [ID!]
 }
 
+input CreatePoolAdvertisementInput {
+    key: KeyString!
+    classifications: ClassificationBelongsToMany!
+}
+
+input UpdatePoolAdvertisementInput {
+    # Pool name and classification
+    name: LocalizedStringInput
+    classifications: ClassificationBelongsToMany!
```

Comment:
Is there a reason why this is non-nullable?  It's a bit awkard to have to specify this in the query and I don't thing other update inputs have it non-nullable.

Reply:
No this should be nullable!